src/v2/Apps/Components/AppShell.tsx

```diff
@@ -66,7 +67,12 @@ export const AppShell: React.FC<AppShellProps> = props => {
       </Box>
 
       <NetworkOfflineMonitor />
-      <HorizontalPadding>{showFooter && <Footer />}</HorizontalPadding>
+
+      <Flex backgroundColor="white">
```

Comment:
This should match the white color in our theme: https://github.com/artsy/palette/blob/master/packages/palette/src/themes/v2.tsx#L113